README-CONAN.md

```diff
@@ -66,23 +67,30 @@ $ pip install conan --upgrade
 
 ```bash
 $ conan --version
-Conan version 1.4.1
+Conan version 1.12.3
 ```
 
 <div id="1-3">
 
-### 1.3) Create a build directory<div id="1-3">
+### 1.3) Create a build directory
 
 
-Create a build directory and will run the conan commands:
+Create a build directory where you will configure and build Exiv2:
 
 ```bash
+$ cd $EXIV2_ROOT
 $ mkdir build
 $ cd build
 $ conan profile list
 ```
 
-**IMPORTANT** _**Visual Studio Users**_ require the profile msvc2017Release64 in %HOMEPATH%\.conan\profiles\msvc2017Release64
+<div id="1-4">
+
+### 1.4) Prepare your conan profile
+
+The first time you run a conan command, a default conan profile will be created. You can create more [profiles](https://docs.conan.io/en/latest/reference/commands/misc/profile.html) to handle different compilers, build modes, etc.
+
+**NOTE**: This tutorial assumes that _**Visual Studio Users**_ have a profile msvc2017Release64 like the following one in `%HOMEPATH%\.conan\profiles\msvc2017Release64`:
```

Comment:
It is not actually **required**. In windows you also get a _default_ profile which has good default values. So I changed this to mention that we assume the existence of that file for the rest of the document.